shared/src/endpoints.ts

```diff
@@ -23,6 +23,19 @@ export enum Endpoints {
   deleteComment = 'deleteComment',
 }
 
+export function WithParams(endpoint: EndpointConfig, ...params: string[]): EndpointConfig {
+  let url = endpoint.url
+  const placeholders = url.split("/").filter(token => token.startsWith(":"))
+  for (let index = 0; index < Math.min(params.length, placeholders.length); index++) {
+    url = url.replace(placeholders[index], params[index])
+  }
```

Comment:
since we want to support multiple placeholders (we don't have to though, we can always add this when we need), let's check we have the right number of args, and throw if not.